ConcernsCaseWork/ConcernsCaseWork/Pages/Home.cshtml

```diff
@@ -67,7 +67,7 @@
 						<td class="govuk-table__cell">
 							@caseModel.TrustNameTitle
 						</td>
-						<td class="govuk-table__cell">
+						<td class="govuk-table__cell" data-sort-value="@caseModel.CaseType">
```

Comment:
Hey Elijah, do we need the data sort value ? I reckon when we did trust / academy we didn't add the sort and only the header.